app/assets/src/components/visualizations/table/Table.jsx

```diff
@@ -0,0 +1,151 @@
+import React from "react";
```

Comment:
Hey Tiago! This class is beautifully written. However, I think one of the reasons we wanted to adopt `react-virtualized` is so we can have fixed headers and a fixed left column while we do both horizontal and vertical scrolling, as in this example: https://bvaughn.github.io/react-virtualized/#/components/ScrollSync

I'm not sure this kind of thing is possible with `<Table>`, and we might have to wrap `<Grid>` directly.

This example uses `<ScrollSync>` with four `<Grids>`: https://github.com/bvaughn/react-virtualized/blob/master/source/ScrollSync/ScrollSync.example.js

Unfortunately this is substantially more complicated, so will let you make the decision about prioritization.

Reply:
As discussed offline, I will still try to make it work with the table later within the data discovery scope and before adding further features to Table.jsx. If unsuccessful, I will convert it to grid.